packages/frontend/src/components/wallet-migration/modals/WalletSelectorModal/WalletSelectorExportContext.jsx

```diff
@@ -0,0 +1,89 @@
+
+import { setupExportSelectorModal } from '@near-wallet-selector/account-export';
+import { setupWalletSelector } from '@near-wallet-selector/core';
+import { setupMeteorWallet } from '@near-wallet-selector/meteor-wallet';
+import { setupMyNearWallet } from '@near-wallet-selector/my-near-wallet';
+import React, { useCallback, useContext, useEffect, useState } from 'react';
+
+import '@near-wallet-selector/modal-ui/styles.css';
+import '@near-wallet-selector/account-export/styles.css';
+import './WalletSelectorModalContext.css';
+
+const ExportAccountSelectorContext =
+  React.createContext(null);
+
+
+// If target wallet is production ready, add it to this list
+const MAINNET_MODULES = [];
+
+const TESTNET_MODULES = [
+    setupMyNearWallet(),
```

Comment:
Is it necessary to invoke these in module scope? Seems like it would be better to invoke these dynamically, i.e. an `initializeModules()` function that takes `network` as a parameter and returns the appropriate set of initialized modules. Then we don't have to worry about potential side effects across networks.